modules/core/src/main/kotlin/com/github/l34130/mise/core/setting/MiseSettings.kt

```diff
@@ -49,7 +52,7 @@ class MiseSettings : PersistentStateComponent<MiseSettings.MyState> {
                     NotificationAction.createSimple(
                         "Open settings",
                     ) {
-                        ShowSettingsUtil.getInstance().showSettingsDialog(null, MiseConfigurable::class.java)
+                        ShowSettingsUtil.getInstance().showSettingsDialog(project, "Mise Settings")
```

Comment:
The root cause that makes `Cannot find configurable for specified predicate` error